packages/core/src/components/portal/_portal.scss

```diff
@@ -34,7 +34,3 @@ The children of a `Portal` component are appended to the `<body>` element.
 
 Styleguide components.portal.js
 */
-
-.pt-portal {
-  display: inline-block;
-}
```

Comment:
actually it does cuz they use the `div` tag which is implicitly `display: block`.